tests/pkg/metrics/annotation_util.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"sync"
+)
+
+//Client request grafana API on a set of resource paths.
+type client struct {
+	// base is the root URL for all invocations of the client
+	baseUrl url.URL
+	client  *http.Client
+}
+
+//Annotation is a specification of the desired behavior of adding annotation
+type Annotation struct {
+	DashboardId         int      `json: "dashboardId, omitempty"`
+	PanelId             int      `json: "panelId, omitempty"`
+	TimestampInMilliSec int64    `json: "time""`
+	Tags                []string `json: "tags, omitempty""`
+	Text                string   `json: "text""`
+	IsRegin             bool     `json: "isRegion, omitempty""`
+	TimeEnd             int64    `json: "timeEnd, omitempty""`
```

Comment:
Typo in above 4 lines, double quote not closed

Reply:
Ha， I just commit it for updating grafana pull request.
This change do not finished
Thanks for your help.